src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Services/CohortDomainServiceTests.cs

```diff
@@ -121,6 +124,24 @@ public async Task OverlapOnEndDate_Validation()
             _fixture.VerifyOverlapExceptionOnEndDate();
         }
 
+        [TestCase(Party.Provider)]
+        [TestCase(Party.Employer)]
+        public async Task UpdateDraftApprenticeship_IsSuccessful_ThenDraftApprenticeshipIsUpdatedAndLastUpdatedFieldsAreSet(Party withParty)
+        {
+            _fixture.WithParty(withParty).WithExistingCohort(withParty).WithExistingDraftApprenticeship();
+            await _fixture.UpdateDraftApprenticeship();
+            _fixture.VerifyDraftApprenticeshipUpdated();
+            _fixture.VerifyLastUpdatedFieldsAreSet(withParty);
```

Comment:
Is it worth moving this into a test of its own, similar to the "Not Set" test below? e.g. `UpdateDraftApprenticeship_WhenUserInfoExists_ThenDraftApprenticeshipLastUpdatedFieldsAreSet`